go.sum

```diff
@@ -0,0 +1,6 @@
+github.com/gorilla/feeds v0.0.0-20170611032206-fa8f5548eedb h1:ry27SfzOHgpUX1TXYh1W/kOUM7J+K8MdpRygT0DtHy0=
```

Comment:
this was automatically generated? (I don't know `go` 😹)

Reply:
Yes! It is generated. [FAQs — go.mod and go.sum](https://github.com/golang/go/wiki/Modules#is-gosum-a-lock-file-why-does-gosum-include-information-for-module-versions-i-am-no-longer-using)